ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -0,0 +1,174 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Messages::TabPageHeader
+  @authenticated={{@authenticated}}
+  @pageTitle="{{if @message.isNew 'Create' 'Edit'}} message"
+  @breadcrumbs={{this.breadcrumbs}}
+/>
+
+<form id="message-create-edit-form" {{on "submit" (perform this.save)}} data-test-form="create-and-edit">
+  <div class="box is-sideless is-fullwidth is-marginless has-top-padding-s">
+    <Hds::Text::Body @tag="p" class="has-bottom-margin-l" data-test-form-subtext>
+      {{if @message.isNew "Create" "Edit"}}
+      a custom message for all users when they access a Vault system via the UI.
+    </Hds::Text::Body>
+
+    <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
+
+    {{#each @message.formFields as |attr|}}
+      {{#if (eq attr.name "authenticated")}}
```

Comment:
I'm wondering whether or not `FormField` should be used for simplicity and then this should all update to HDS components automatically when we start updating that component to HDS?

Reply:
@zofskeez I was wondering that too! The issue is that the design has a title and subtext explaining each radio button, which I don't think we have yet. I wasn't sure we adding some extra CSS is necessary since we'll eventually use HDS. Thoughts? I think it would be cleaner to use the FormField radio as well!